devdox/tests/unit_test/app/test_main.py

```
@@ -0,0 +1,48 @@
+    def __init__(self):
+        self.closed = False
+    def close(self):
+        self.closed = True
+
+class DummyMongoDb:
+    async def command(self, cmd):
+        raise Exception("boom")
+
+@pytest.mark.asyncio
+async def test_init_mongo_closes_client_on_failed_ping(monkeypatch, mongo_settings):
+    async def fake_init_via_uri(uri):
```

Ruff (0.14.7)

Unused function argument: `uri`

(ARG001)